module/LineTracer.cpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file LineTracer.cpp
+ * @brief ライントレースする
+ * @author yutotanaka24,kodama0720
+ */
+#include "LineTracer.h"
+
+LineTracer::LineTracer(bool _isLeft) : edge((_isLeft == true) ? 1 : -1) {}
+
+//設定された距離を走行する
+void LineTracer::run(double targetDistance, int targetBrightness, int pwm, PidGain& gain)
```

Comment:
```suggestion
void LineTracer::run(double targetDistance, int targetBrightness, int pwm, const PidGain& gain)
```
僕がこの関数を呼び出す際に、constな構造体のメンバ変数を実引数に渡すので、参照型の仮引数にはconstをつけておかないとエラーが出たりする気がします。言い忘れてて申し訳ないです。